Sources/SwiftCompilerPluginMessageHandling/CompilerPluginMessageHandler.swift

```diff
@@ -30,6 +30,9 @@ import SwiftSyntaxMacros
 @_spi(PluginMessage)
 public enum PluginFeature: String {
   case loadPluginLibrary = "load-plugin-library"
+
+  /// Whether the plugin knows how to infer nonisolated conformances.
+  case inferNonisolatedConformances = "infer-nonisolated-conformances"
```

Comment:
This doesn't seem to be used.

Reply:
It ends up getting passed back to the compiler, which checks for its presence. See the [compiler PR](https://github.com/swiftlang/swift/pull/81260) I just put up.

Reply:
Okay, but I consider `PluginFeature` is more like variation of plugin executable. E.g. single-module-plugin v.s. plugin-server. For something like this, I'd use `PluginMessage.PROTOCOL_VERSION_NUMBER`. Like, in `PluginHost.swift` in the swift repo: 
```swift
extension CompilerPlugin {
  hasIinferNonisolatedConformances: Bool { capability.protcolVersion >= 8 }
}
```

Also, you only added this feature to `LibraryPluginProvider`, which doesn't cover single-module executable  plugins.